build.zig

```diff
@@ -38,29 +39,21 @@ pub fn build(b: *Builder) void {
     }
 
     if (examples) {
-        const globals = b.addExecutable("globals", "example/globals.zig");
-        globals.setTarget(target);
-        globals.setBuildMode(mode);
+        const example_names = [_][]const u8{ "globals", "listener", "seats" };
+        for (example_names) |example| {
+            const path = mem.concat(b.allocator, u8, &[_][]const u8{ "example/", example, ".zig" }) catch @panic("sdf");
```

Comment:
Catch unreachable is fine in `build.zig`s and is probably better than a random panic message.
```suggestion
            const path = mem.concat(b.allocator, u8, &[_][]const u8{ "example/", example, ".zig" }) catch unreachable;
```